projects/core/src/checkout/facade/checkout-delivery.service.ts

```diff
@@ -34,7 +36,8 @@ export class CheckoutDeliveryService {
   constructor(
     protected checkoutStore: Store<StateWithCheckout | StateWithProcess<void>>,
     protected activeCartService: ActiveCartService,
-    protected userIdService: UserIdService
+    protected userIdService: UserIdService,
+    protected checkoutService: CheckoutService
   ) {}
 
```

Comment:
It would be good to notify the customer about this change in case they are extending this service, so that they can pass it to `super()` call.
The best way to do this would be to update the 3.4 docs, but also write this explanation in the body of this PR, and just copy it to the merge commit when merging. This is useful for clients who will go to our GH and check recent commits.